### ISSUE TITLE
make build failes for newer docker version > 24

### ISSUE THREAD

**Issue by avarei** [crossplane/build#8]
When running `make build` in a repo with this submodule I get.

```
15:31:25 [ OK ] docker build build-7196b0f9/provider-vra-arm64
15:31:25 [ .. ] Building package provider-vra-v0.0.0-38.g1d7f71f.dirty.xpkg for linux_arm64
up: error: xpkg.buildCmd.Run(): unable to calculate manifest: blob sha256:4f97f5713e2bdd62934f95548d404be13994984ccfacb779722fc5fef6060b53 not found
15:31:26 [FAIL]
```

[This Slack message](https://crossplane.slack.com/archives/CEG3T90A1/p1718267967283659) describes the same issue and they solved it by downgrading to docker 24.

This worked for me as well as a workaround but I hope this can be fixed.

A lot of Provider Pipelines that rely on this submodule still use an old version of Buildx that is compatible. They should continue to work with the new solution.

**Comment by jbw976** [comment]
Yep, I think this issue should be resolved for folks using newer versions of this build submodule which use the Crossplane CLI instead of the `up` CLI by default. Just make sure your Makefile specifies `$(CROSSPLANE_CLI)` as a target dependency so it will be downloaded and ready for usage in the build process as @chlunde pointed out 🎉
